docs/azure_arc_jumpstart/azure_arc_k8s/aksengine/aksengine_azcli/_index.md

```diff
@@ -0,0 +1,117 @@
+
+  ```shell
+  az --version
+  ```
+
+* [Generate SSH Key](https://docs.microsoft.com/en-us/azure/virtual-machines/linux/create-ssh-keys-detailed) (or use existing ssh key).
+
+* Create Azure service principal (SP)
+
+    To be able to complete the scenario and its related automation, Azure service principal assigned with the “Contributor” role is required. To create it, login to your Azure account run the below command (this can also be done in [Azure Cloud Shell](https://shell.azure.com/)).
+
+    ```shell
+    az login
+    az ad sp create-for-rbac -n "<Unique SP Name>" --role contributor
+    ```
+
+    For example:
+
+    ```shell
+    az ad sp create-for-rbac -n "http://AzureArcK8s" --role contributor
+    ```
+
+    Output should look like this:
+
+    ```json
+    {
+    "appId": "XXXXXXXXXXXXXXXXXXXXXXXXXXXX",
+    "displayName": "AzureArcK8s",
+    "name": "http://AzureArcK8s",
+    "password": "XXXXXXXXXXXXXXXXXXXXXXXXXXXX",
+    "tenant": "XXXXXXXXXXXXXXXXXXXXXXXXXXXX"
+    }
+    ```
+
+    > **Note: The Jumpstart scenarios are designed with as much ease of use in-mind and adhering to security-related best practices whenever possible. It is optional but highly recommended to scope the service principal to a specific [Azure subscription and resource group](https://docs.microsoft.com/en-us/cli/azure/ad/sp?view=azure-cli-latest) as well considering using a [less privileged service principal account](https://docs.microsoft.com/en-us/azure/role-based-access-control/best-practices)**
+## Installing AKS Engine
+
+* AKS Engine requires a binary installed in the system. You can refer to [AKS Engine Releases](https://github.com/Azure/aks-engine/releases/latest) to get the latest version, the following example downloads and installs version 0.61.0:
+
+```bash
+# Install AKS engine
+# Go here: https://github.com/Azure/aks-engine/releases/latest
+# Example with v0.61.0:
+aksengine_exec=$(which aks-engine)
+if [[ -n "$aksengine_exec" ]]
+then
+    echo "aks-engine executable found in ${aksengine_exec}"
+else
+    echo "Downloading and installing aks-engine executable..."
+    aksengine_tmp=/tmp/aksengine.tar.gz
+    wget https://github.com/Azure/aks-engine/releases/download/v0.61.0/aks-engine-v0.61.0-linux-amd64.tar.gz -O $aksengine_tmp
+    tar xfvz $aksengine_tmp -C /tmp/
+    sudo cp /tmp/aks-engine-v0.61.0-linux-amd64/aks-engine /usr/local/bin
+fi
+```
+
+## Deployment
+
+In this guide we will generate an AKS Engine cluster with 1 master node and 2 worker nodes. There are additional examples for the required JSON files with different configurations in [AKS Engine Examples](https://github.com/Azure/aks-engine/tree/master/examples). 
+
+* Before initiating the deployment, edit the [*arc_k8s_aksengine.sh script*](https://azurearcjumpstart.io/azure_arc_jumpstart/azure_arc_k8s/azure_arc_k8s_jumpstart/aksengine/arc_k8s_aksengine.sh) script so that the environment variables match your setup:
```

Comment:
```suggestion
* Before initiating the deployment, edit the [*arc_k8s_aksengine.sh script*](https://github.com/erjosito/azure_arc/blob/aksengine/azure_arc_k8s_jumpstart/main/arc_k8s_aksengine.she.sh) script so that the environment variables match your setup:
```

Also, change the folder name from _aksengine_ to _aks-engine_.